### PR TITLE
Retrieve mr number from commit hash

### PR DESCRIPTION
**Pull request by kitagry**
Hi @yuyaban . Thank you for the nice tool!

I want to comment to MR when the MR is merged. So, I implemented the feature to retrieve MR number associated with SHA1.

And, sorry:pray: This PR include #9 commits. :bow: :bow:
So, I will close #9

### CODE REVIEW

**Comment by yuyaban** [comment]
Thank you for your contribution.
There are two things I would like to see fixed.
If possible, please modified.

**Comment by yuyaban** [comment]
Currently, `getMRNumber` returns Error if `CI_MERGE_REQUEST_IID` is not found.
Since `CI_MERGE_REQUEST_IID` is not defined at merging MRs pipeline, I think it is better to return `nil` if `CI_MERGE_REQUEST_IID` is not found.
https://github.com/yuyaban/gitlab-comment/blob/469c9a85038e7e42e88be68dc30e03c361d20b69/pkg/platform/platform.go#L37-L46

File: pkg/gitlab/mr.go [hunk 1]
```diff
@@ -5,19 +5,17 @@ import (
...
-	// if len(prs) == 0 {
-	// 	return 0, errors.New("associated merge request isn't found")
-	// }
-	// return prs[0].GetNumber(), nil
+	mrList, _, err := client.commit.ListMergeRequestsByCommit(
+		fmt.Sprintf("%s/%s", owner, repo),
+		sha,
+	)
+	if err != nil {
+		return 0, err
```
Comment (yuyaban) [inline pkg/gitlab/mr.go]: The test is failed below.
So, should be modified as shown in the Suggestions.
```suggestion
		return 0, fmt.Errorf("list associated pull requests: %w", err)
```

**Comment by kitagry** [comment]
@yuyaban Thank you for the review! I fixed

**Comment by yuyaban** [comment]
@kitagry Thank you for your contribution!! 🙇

**Pull request merged**